app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
```

Comment:
Do not create these attributes in advance, you should add `wife` and `husband` attributes to `Person` class instances dynamically depending on the data provided in the `people` list in the `create_person_list` method.